### DISCUSSION TITLE
iOS/ iPad RustDesk client no showing Host ID

### DISCUSSION THREAD

**Discussion by kilian099** [rustdesk/rustdesk#7145] [Q&A]
Hi Team,
I was trying to set it up in an iPad RustDesk Client app, and it was suspicious I could not get my host ID to provide to the remote device to connect to my device. Do you know if anyone has set up the iPad RustDesk app before? Thanks.
Kilian
![WhatsApp Image 2024-02-15 at 13 27 58_acdeb857](https://github.com/rustdesk/rustdesk/assets/106171763/e53f6aa6-7c5b-4c5b-a05c-51c59ef94ca6)

**Comment by tschettervictor** [comment]
I have it working. There are some bugs still though. One of which is what you said. Discovery doesn’t seem to work properly.

You can still type the ID and connect.

**Reply by charettepa** [reply]
> I have it working. There are some bugs still though. One of which is what you said. Discovery doesn’t seem to work properly.
> 
> You can still type the ID and connect.

yes I am also trying to type the ID
but what am i supposed to type if i dont know the ID

I dont want to connect from it
I want to connect to it

**Reply by tschettervictor** [reply]
You can't connect to iOS at the moment. I know teamviewer abs anydesk allow this, but rustdesk has not implemented it.

And even if they would, it works only allow viewing and not controlling.

**Reply by charettepa** [reply]
viewing is all i need. the person I'm helping is not the brightest

**Reply by tschettervictor** [reply]
Understood. I guess we will have to wait and see if this feature gets implemented.

You can vote for it in their discussion page.

**Comment by BiswajitRedoQ** [comment]
Hi,
Also I am facing the same issue. Is there any update on the '**fixing**' of that issue or you have any updated solution regarding that issue?

**Comment by CyBuzz** [comment]
I am just posting to say that I am experiencing the same behavior. I can connect from my iPad to my Linux box but not the other way around because I don’t have a host id for my iPad.

**Reply by CyBuzz** [reply]
If I connect to my Linux box, it appears that the host id is displayed in the connection box. Figured that out using my android phone connecting to Linux.  That being said, it appears that I dont have or cannot start the server portion of rust desk on my iPad.

**Comment by MichiFr** [comment]
Same here on iPadOS, self hosted rust server in Docker container. I, too, do not see my local ID to connect to from e.g. Windows.

Btw, the docker is running as an LXC on my Proxmox, so docker has got it's own IP address, different from the Proxmox server.

Which IP address will you use for the ID server? The docker's or the Proxmox' one?

**Comment by wzxklm** [comment]
The Apple system cannot be controlled
